portfolio/src/main/webapp/js/background.js

```diff
@@ -57,9 +71,10 @@ function initTank() {
  * https://p5js.org/reference/#/p5/windowResized
  */
 function windowResized() {
-  const canvasContainer = document.getElementById("background-container");
-  resizeCanvas(canvasContainer.clientWidth,
-    canvasContainer.clientWidth * HEIGHT_TO_WIDTH);
+  const backgroundContainer = document.getElementById("background-container");
+  const tankHeightToWidth = TANK_HEIGHT / TANK_WIDTH;
+  resizeCanvas(backgroundContainer.clientWidth,
+    backgroundContainer.clientWidth * tankHeightToWidth);
```

Comment:
```suggestion
      backgroundContainer.clientWidth * tankHeightToWidth);
```
https://google.github.io/styleguide/jsguide.html#formatting-function-arguments "Indentation should be four spaces"